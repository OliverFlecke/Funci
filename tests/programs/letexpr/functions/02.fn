-- make sure they bind correctly

main :: Int =
    let x :: Int = 2
    in let square x :: Int -> Int = x * x
    in square (square x)