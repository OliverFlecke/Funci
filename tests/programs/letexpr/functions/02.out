Number (I 16) (Unit [])